### DISCUSSION TITLE
When comparing folders is it possible to display the size of files?

### DISCUSSION THREAD

**Discussion by ZENAdmin-Ops** [WinMerge/winmerge#2209] [Q&A] [answered]
Hello,

I'm comparing two folders and I'm interested in the differences between the two folders.

In particular the larger files that have changed.

I'm currently using the following view settings

Show Different Items
Show Left Unique
Show Right Unique
Show Binary Files

Is there a way that I can include the file size in the comparison result?

Thank you.

**Comment by sdottaka** [comment, marked as answer]
Selecting Left Size and Right Size in the Display Columns window adds Left File Size and Right File Size columns, as shown below.

![image](https://github.com/WinMerge/winmerge/assets/98126/980bc886-1438-442a-8f5c-051834ba519a)
![image](https://github.com/WinMerge/winmerge/assets/98126/7a47b7ed-dd49-4481-be69-aa5f8e1da199)

**Comment by ZENAdmin-Ops** [comment]
Thanks!
